### PR TITLE
Fixing import errors

### PR DESCRIPTION
**Pull request by nekiter**
Fixes #

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

On the [PR #353](https://github.com/HTTP-APIs/hydrus/commit/525b694f200ad87855327af24e8d1426078b0989) of hydrus repo, was made a port hydraspec to hydra_python_core and that made this repo outdate.

#### Fixed
This PR solves the error:

ModuleNotFoundError: No module named 'hydrus.hydraspec'

Now it's solved.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 requests
 haversine
 httplib2
--e git://github.com/pchampin/hydra-py.git#egg=hydra
--e git+https://github.com/HTTP-APIs/hydrus.git#egg=hydrus
+-e git://github.com/HTTP-APIs/hydra-python-core.git#egg=hydra_python_core
```
Comment (Mec-iS) [inline requirements.txt]: We don't need the `-e` flag. Please substitute these lines by using the new approach as displayed [here](https://github.com/HTTP-APIs/hydrus/blob/develop/requirements.txt#L26)

**Comment by Mec-iS** [comment]
Base branch for PR is always `develop`

**Comment by Mec-iS** [comment]
Number #17 should be merged in this one.

**Comment by xadahiya** [comment]
@nekiter  ^^

**Comment by Mec-iS** [comment]
@nekiter you have a related PR at https://github.com/nekiter/hydra-flock-central-controller/pull/1

**Pull request closed without merging**